plasma_framework/contracts/src/framework/utils/Quarantine.sol

```diff
@@ -0,0 +1,33 @@
+pragma solidity ^0.5.0;
+
+/**
+ * @title Provides a way to quarantine (disable) contracts for a period of time
+ * @dev The immunitiesRemaining member allows us to deploy the platform with some
+ * pre-verified contracts that don't get quarantined.
+ */
+library Quarantine {
```

Comment:
I think we should use `internal` instead of `public` for functions here. This reduce some gas to external contract and we also don't need to link anymore. (though it increases contract deployment cost)